packages/components/src/components/hds/icon-tile/types.ts

```diff
@@ -0,0 +1,44 @@
+export enum HdsIconTileSizeValues {
+  Small = 'small',
+  Medium = 'medium',
+  Large = 'large',
+}
+
+export type HdsIconTileSizes = `${HdsIconTileSizeValues}`;
+
+export enum HdsIconTileProductValues {
+  Boundary = 'boundary',
+  Consul = 'consul',
+  HCP = 'hcp',
+  Nomad = 'nomad',
+  Packer = 'packer',
+  Terraform = 'terraform',
+  Vagrant = 'vagrant',
+  Vault = 'vault',
+  'Vault Secrets' = 'vault-secrets',
+  'Vault Radar' = 'vault-radar',
+  Waypoint = 'waypoint',
+}
+
+export type HdsIconTileProducts = `${HdsIconTileProductValues}`;
+
+export enum HdsIconTileColorNeutral {
+  Neutral = 'neutral',
+}
+
+type HdsIconTileColorValues =
```

Comment:
```suggestion
export type HdsIconTileColorValues =
```

Reply:
actually, that isn't quite it...

Reply:
indeed, the [initial suggestion](https://github.com/hashicorp/design-system/pull/2033#discussion_r1551414299) addressed the type, but not the array of colours we use for validation and in our showcase app. it's rather tricky to properly extend enums, so I'd use the same approach we used for types and amend 'neutral' to product values. pushed a commit for illustration

Reply:
Thanks! TIL!